include/clang/AST/Expr.h

```diff
@@ -4814,13 +4814,24 @@ class RangeBoundsExpr : public BoundsExpr {
 private:
   enum { LOWER, UPPER, END_EXPR };
   Stmt *SubExprs[END_EXPR];
+  RelativeBoundsClause *DefaultClause;
   RelativeBoundsClause *RelativeClause;
+  /// Relative alignment base : bounds(lb, ub) rel_align(T)
+  /// compiler checkes whether sizeof(referent-type(base)) is common factor of
```

Comment:
The semantics is slightly different: 
Normally, the compiler assumes that ub-lb is a multiple of sizeof(referentype(base)).
This lets the compiler generate more efficient code for bounds checks.  With
a rel_align clause, the compiler assume that ub-lb is a multiple of sizeof(T).